### ISSUE TITLE
Update 14.1.0 causes Server error

### ISSUE THREAD

**Issue by Praetorian1410** [nextcloud/bookmarks#2165]
**Describe the bug**
After performing the latest update of the bookmarks app (14.1.0), the entire cloud refuses to load with a server error: Request ID: qaDlcbTM0O41Y5FNqQOc
Disabling bookmarks via OCC allows the cloud to start again.
Reenabling boomarks again causes the issue again.

**To Reproduce**
1. Update bookmarks app to latest version
2. Refresh page
3. Observe error

**Expected behavior**
Bookmarks to not break Nextcloud

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Firefox
 - Version: 126.0

**Server (please complete the following information):**
- Installation type: Nextcloud AIO (Docker, Raspberry Pi 5)
- Nextcloud version: 29.0.0
- PHP Version: 8.2.18
- Database: PostgreSQL 15.6 on aarch64
- Bookmarks app version: 14.1.0
- Activated Nextcloud Apps:
 - Enabled:
  - activity: 2.21.1
  - bookmarks: 14.1.0
  - bruteforcesettings: 2.9.0
  - calendar: 4.7.4
  - calendar_news: 1.1.12
  - circles: 29.0.0-dev
  - cloud_federation_api: 1.12.0
  - collectives: 2.11.0
  - comments: 1.19.0
  - contacts: 6.0.0
  - cookbook: 0.11.0
  - cospend: 1.6.1
  - dashboard: 7.9.0
  - dav: 1.30.1
  - federatedfilesharing: 1.19.0
  - federation: 1.19.0
  - files: 2.1.0
  - files_downloadlimit: 2.0.0
  - files_external: 1.21.0
  - files_pdfviewer: 2.10.0
  - files_reminders: 1.2.0
  - files_sharing: 1.21.0
  - files_trashbin: 1.19.0
  - files_versions: 1.22.0
  - integration_openstreetmap: 1.0.12
  - integration_paperless: 1.0.3
  - integration_tmdb: 1.0.4
  - integration_youtube: 0.2.1
  - logreader: 2.14.0
  - lookup_server_connector: 1.17.0
  - maps: 1.4.0
  - memories: 7.3.1
  - news: 25.0.0-alpha6
  - nextcloud-aio: 0.5.0
  - notes: 4.10.0
  - notifications: 2.17.0
  - notify_push: 0.6.12
  - oauth2: 1.17.0
  - oidc: 0.9.0
  - otpmanager: 0.5.3
  - password_policy: 1.19.0
  - passwords: 2024.5.20
  - photos: 2.5.0
  - picker: 1.0.8
  - previewgenerator: 5.5.0
  - provisioning_api: 1.19.0
  - recognize: 7.0.0
  - recommendations: 2.1.0
  - richdocuments: 8.4.2
  - serverinfo: 1.19.0
  - settings: 1.12.0
  - sharebymail: 1.19.0
  - side_menu: 3.12.0
  - spreed: 19.0.1
  - support: 1.12.0
  - survey_client: 1.17.0
  - tables: 0.7.2
  - text: 3.10.0
  - theming: 2.4.0
  - theming_customcss: 1.16.0
  - twofactor_backupcodes: 1.18.0
  - twofactor_nextcloud_notification: 3.9.0
  - twofactor_totp: 11.0.0-dev
  - unroundedcorners: 1.1.3
  - viewer: 2.3.0
  - weather_status: 1.9.0
  - workflowengine: 2.11.0
Disabled:
  - admin_audit: 1.19.0 (installed 1.19.0)
  - contactsinteraction: 1.10.0 (installed 1.10.0)
  - encryption: 2.17.0
  - files_inotify: 0.2.1
  - firstrunwizard: 2.18.0 (installed 2.18.0)
  - nextcloud_announcements: 1.18.0 (installed 1.18.0)
  - privacy: 1.13.0 (installed 1.13.0)
  - related_resources: 1.4.0 (installed 1.4.0)
  - suspicious_login: 7.0.0
  - systemtags: 1.19.0 (installed 1.19.0)
  - user_ldap: 1.20.0
  - user_status: 1.9.0 (installed 1.9.0)

- Nextcloud configuration: 
 ` {
    "system": {
        "one-click-instance": true,
        "one-click-instance.user-limit": 100,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "appsallowlist": false,
        "check_data_directory_permissions": false,
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "password": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "overwritehost": "cloud.praetos.dnsuser.de",
        "overwriteprotocol": "https",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "cloud.praetos.dnsuser.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "29.0.0.19",
        "overwrite.cli.url": "https:\/\/cloud.praetos.dnsuser.de\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "updatedirectory": "\/nc-updater",
        "loglevel": "2",
        "app_install_overwrite": [
            "nextcloud-aio",
            "news",
            "calendar_news",
            "maps",
            "otpmanager"
        ],
        "log_type": "file",
        "logfile": "\/var\/www\/html\/data\/nextcloud.log",
        "log_rotate_size": "10485760",
        "log.condition": {
            "apps": [
                "admin_audit"
            ]
        },
        "preview_max_x": 2048,
        "preview_max_y": 2048,
        "jpeg_quality": 60,
        "enabledPreviewProviders": [
            "OC\\Preview\\Imaginary",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\Movie",
            "OC\\Preview\\Krita"
        ],
        "enable_previews": true,
        "upgrade.disable-web": true,
        "mail_smtpmode": "smtp",
        "trashbin_retention_obligation": "auto, 30",
        "versions_retention_obligation": "auto, 30",
        "activity_expire_days": "30",
        "simpleSignUpLink.shown": false,
        "share_folder": "\/Shared",
        "one-click-instance.link": "https:\/\/nextcloud.com\/all-in-one\/",
        "upgrade.cli-upgrade-link": "https:\/\/github.com\/nextcloud\/all-in-one\/discussions\/2726",
        "maintenance_window_start": 100,
        "allow_local_remote_servers": true,
        "davstorage.request_timeout": 3600,
        "htaccess.RewriteBase": "\/",
        "dbpersistent": false,
        "auth.bruteforce.protection.enabled": true,
        "ratelimit.protection.enabled": true,
        "files_external_allow_create_new_local": true,
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "preview_imaginary_url": "***REMOVED SENSITIVE VALUE***",
        "preview_imaginary_key": "d01bb0215c5958a80a851f06a6a14cbc38f8cb6850162a37",
        "memories.db.triggers.fcu": true,
        "memories.exiftool": "\/var\/www\/html\/custom_apps\/memories\/bin-ext\/exiftool-aarch64-musl",
        "memories.vod.path": "\/var\/www\/html\/custom_apps\/memories\/bin-ext\/go-vod-aarch64",
        "memories.vod.ffmpeg": "\/usr\/bin\/ffmpeg",
        "memories.vod.ffprobe": "\/usr\/bin\/ffprobe",
        "memories.viewer.high_res_cond_default": "always",
        "memories.gis_type": 2,
        "default_phone_region": "DE",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpport": "465",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***"
    }
}
`
 - Nextcloud external user backend: none

**Web server error log**
```
None
```

**Nextcloud log (nextcloud/data/nextcloud.log)**
```
None
```

**Nextcloud AIO Container log (might not be related)**
```
Line: 76
File: /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php
Message: Not found.
4
#12 /var/www/docker-aio/php/public/index.php(185): Slim\App->run()
#13 {main}
Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.
#2 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
Trace: #0 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(56): Slim\Middleware\RoutingMiddleware->performRouting(Object(GuzzleHttp\Psr7\ServerRequest))
#1 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(482): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#3 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#5 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#7 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#9 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#10 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#11 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
```

**Browser log**
```
Nothing visibly relevant (can attach if requested anyway)
```

**Comment by koying** [comment]
Same. Saw this in the log that might be related

```
{"reqId":"ueGqHvyWBQOK04kZNwXL","level":3,"time":"2024-06-01T11:38:10+00:00","remoteAddr":"172.71.182.158","user":"koying","app":"webdav","method":"PROPFIND","url":"/remote.php/webdav/","message":"Class \"OCA\\Circles\\Model\\Circle\" not found","userAgent":"davfs2/1.5.5 neon/0.30.2","version":"29.0.1.1","exception":{"Exception":"Error","Message":"Class \"OCA\\Circles\\Model\\Circle\" not found","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":1028,"function":"getSingleCircle","class":"OCA\\Circles\\Db\\CircleRequest","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":995,"function":"getSingleCircle","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":543,"function":"fillSingleCircleId","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":738,"function":"getLocalFederatedUser","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/CirclesManager.php","line":141,"function":"getFederatedUser","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":44,"function":"getFederatedUser","class":"OCA\\Circles\\CirclesManager","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":64,"function":"getFederatedUser","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":85,"function":"startSession","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CollectiveHelper.php","line":21,"function":"getCircles","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Mount/MountProvider.php","line":38,"function":"getCollectivesForUser","class":"OCA\\Collectives\\Service\\CollectiveHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Mount/MountProvider.php","line":86,"function":"getFoldersForUser","class":"OCA\\Collectives\\Mount\\MountProvider","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Config/MountProviderCollection.php","line":89,"function":"getMountsForUser","class":"OCA\\Collectives\\Mount\\MountProvider","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Config/MountProviderCollection.php","line":138,"function":"getMountsFromProvider","class":"OC\\Files\\Config\\MountProviderCollection","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":221,"function":"addMountForUser","class":"OC\\Files\\Config\\MountProviderCollection","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":320,"function":"OC\\Files\\{closure}","class":"OC\\Files\\SetupManager","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":222,"function":"setupForUserWith","class":"OC\\Files\\SetupManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Filesystem.php","line":362,"function":"setupForUser","class":"OC\\Files\\SetupManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/Cache/File.php","line":56,"function":"initMountPoints","class":"OC\\Files\\Filesystem","type":"::"},{"file":"/var/www/nextcloud/lib/private/Cache/File.php","line":177,"function":"getStorage","class":"OC\\Cache\\File","type":"->"},{"file":"/var/www/nextcloud/lib/base.php","line":884,"function":"gc","class":"OC\\Cache\\File","type":"->"},{"function":"{closure}","class":"OC","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Hooks/EmitterTrait.php","line":105,"function":"call_user_func_array"},{"file":"/var/www/nextcloud/lib/private/Hooks/PublicEmitter.php","line":40,"function":"emit","class":"OC\\Hooks\\BasicEmitter","type":"->"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":412,"function":"emit","class":"OC\\Hooks\\PublicEmitter","type":"->"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":684,"function":"completeLogin","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":364,"function":"loginWithToken","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":463,"function":"login","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":113,"function":"logClientIn","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Backend/AbstractBasic.php","line":103,"function":"validateUserPass","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":231,"function":"check","class":"Sabre\\DAV\\Auth\\Backend\\AbstractBasic","type":"->"},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":138,"function":"auth","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Plugin.php","line":180,"function":"check","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Plugin.php","line":135,"function":"check","class":"Sabre\\DAV\\Auth\\Plugin","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"beforeMethod","class":"Sabre\\DAV\\Auth\\Plugin","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":456,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/apps/dav/appinfo/v1/webdav.php","line":88,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/remote.php","line":172,"args":["/var/www/nextcloud/apps/dav/appinfo/v1/webdav.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/circles/lib/Db/CircleRequest.php","Line":429,"message":"Class \"OCA\\Circles\\Model\\Circle\" not found","exception":{},"CustomMessage":"Class \"OCA\\Circles\\Model\\Circle\" not found"}} 
```

Disabling the Bookmarks app restores NC

**Comment by Praetorian1410** [comment]
> Same. Saw this in the log that might be related
> 
> ```
> {"reqId":"ueGqHvyWBQOK04kZNwXL","level":3,"time":"2024-06-01T11:38:10+00:00","remoteAddr":"172.71.182.158","user":"koying","app":"webdav","method":"PROPFIND","url":"/remote.php/webdav/","message":"Class \"OCA\\Circles\\Model\\Circle\" not found","userAgent":"davfs2/1.5.5 neon/0.30.2","version":"29.0.1.1","exception":{"Exception":"Error","Message":"Class \"OCA\\Circles\\Model\\Circle\" not found","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":1028,"function":"getSingleCircle","class":"OCA\\Circles\\Db\\CircleRequest","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":995,"function":"getSingleCircle","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":543,"function":"fillSingleCircleId","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/Service/FederatedUserService.php","line":738,"function":"getLocalFederatedUser","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/circles/lib/CirclesManager.php","line":141,"function":"getFederatedUser","class":"OCA\\Circles\\Service\\FederatedUserService","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":44,"function":"getFederatedUser","class":"OCA\\Circles\\CirclesManager","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":64,"function":"getFederatedUser","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CircleHelper.php","line":85,"function":"startSession","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Service/CollectiveHelper.php","line":21,"function":"getCircles","class":"OCA\\Collectives\\Service\\CircleHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Mount/MountProvider.php","line":38,"function":"getCollectivesForUser","class":"OCA\\Collectives\\Service\\CollectiveHelper","type":"->"},{"file":"/var/www/nextcloud/apps/collectives/lib/Mount/MountProvider.php","line":86,"function":"getFoldersForUser","class":"OCA\\Collectives\\Mount\\MountProvider","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Config/MountProviderCollection.php","line":89,"function":"getMountsForUser","class":"OCA\\Collectives\\Mount\\MountProvider","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Config/MountProviderCollection.php","line":138,"function":"getMountsFromProvider","class":"OC\\Files\\Config\\MountProviderCollection","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":221,"function":"addMountForUser","class":"OC\\Files\\Config\\MountProviderCollection","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":320,"function":"OC\\Files\\{closure}","class":"OC\\Files\\SetupManager","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Files/SetupManager.php","line":222,"function":"setupForUserWith","class":"OC\\Files\\SetupManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/Files/Filesystem.php","line":362,"function":"setupForUser","class":"OC\\Files\\SetupManager","type":"->"},{"file":"/var/www/nextcloud/lib/private/Cache/File.php","line":56,"function":"initMountPoints","class":"OC\\Files\\Filesystem","type":"::"},{"file":"/var/www/nextcloud/lib/private/Cache/File.php","line":177,"function":"getStorage","class":"OC\\Cache\\File","type":"->"},{"file":"/var/www/nextcloud/lib/base.php","line":884,"function":"gc","class":"OC\\Cache\\File","type":"->"},{"function":"{closure}","class":"OC","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/Hooks/EmitterTrait.php","line":105,"function":"call_user_func_array"},{"file":"/var/www/nextcloud/lib/private/Hooks/PublicEmitter.php","line":40,"function":"emit","class":"OC\\Hooks\\BasicEmitter","type":"->"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":412,"function":"emit","class":"OC\\Hooks\\PublicEmitter","type":"->"},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":684,"function":"completeLogin","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":364,"function":"loginWithToken","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/User/Session.php","line":463,"function":"login","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":113,"function":"logClientIn","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Backend/AbstractBasic.php","line":103,"function":"validateUserPass","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":231,"function":"check","class":"Sabre\\DAV\\Auth\\Backend\\AbstractBasic","type":"->"},{"file":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/Auth.php","line":138,"function":"auth","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Plugin.php","line":180,"function":"check","class":"OCA\\DAV\\Connector\\Sabre\\Auth","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Auth/Plugin.php","line":135,"function":"check","class":"Sabre\\DAV\\Auth\\Plugin","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"beforeMethod","class":"Sabre\\DAV\\Auth\\Plugin","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":456,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/apps/dav/appinfo/v1/webdav.php","line":88,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/var/www/nextcloud/remote.php","line":172,"args":["/var/www/nextcloud/apps/dav/appinfo/v1/webdav.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/circles/lib/Db/CircleRequest.php","Line":429,"message":"Class \"OCA\\Circles\\Model\\Circle\" not found","exception":{},"CustomMessage":"Class \"OCA\\Circles\\Model\\Circle\" not found"}} 
> ```
> 
> Disabling the Bookmarks app restores NC

Sounds like it might be related to the sharing bookmarks via circles feature... didn't see that error myself, might just be blind though - which log was that in? I'm running Nextcloud AIO, went through all the logs I could find (just doublechecked the nextcloud container log, no dice).

**Comment by koying** [comment]
> which log was that in

That's in `nextcloud.log` , in `/var/lib/nextcloud` (by me, not sure what's the default)

**Comment by HenkVirtual** [comment]
The same here, disable the bookmarks app en NC is running again
![Screenshot from 2024-06-01 14-25-16](https://github.com/nextcloud/bookmarks/assets/71253263/80768555-de8d-4874-b702-623858b1002e)

**Comment by marcelklehr** [comment]
Very strange. I'm on it.

**Comment by marcelklehr** [comment]
Should be fixed now, in v14.1.1

**Comment by Praetorian1410** [comment]
> Should be fixed now, in v14.1.1

'tis indeed. Danke für deine Arbeit Marcel!
